src/Servers/Kestrel/test/FunctionalTests/HttpsConnectionAdapterTests.cs

```diff
@@ -418,7 +418,7 @@ public async Task DoesNotSupportTls10()
                 using (var client = new TcpClient())
                 {
                     var stream = await OpenSslStream(client, server);
-                    var ex = await Assert.ThrowsAsync<IOException>(
+                    var ex = await Assert.ThrowsAnyAsync<Exception>(
```

Comment:
What else was being thrown?

Reply:
From previous build ([#20221003.26](https://dev.azure.com/dnceng-public/public/_build/results?buildId=39358&view=results)):

``` text
Expected: typeof(System.IO.IOException)
Actual:   typeof(System.ComponentModel.Win32Exception): The client and server cannot communicate, because they do not possess a common algorithm
---- System.ComponentModel.Win32Exception : The client and server cannot communicate, because they do not possess a common algorithm
```